docs/conceptual/Bar-Memory.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+   :description: Learn about BAR memory and ways to handle physical addressing limit in ROCm
+   :keywords: BAR memory, MMIO, GPU memory, Physical Addressing Limit, AMD, ROCm
+
+******************************
+BAR memory overview
+******************************
+From the BIOS of an Intel Xeon E5-based system, you can enable above 4 GB PCIe addressing. You also need to set the
+memory-mapped input/output (MMIO) High Base address ``MMIOH Base`` and range ``MMIO High Size`` in the BIOS.
+
+In the Supermicro system, under the system BIOS, go to ``Advanced > PCIe/PCI/PnP configuration``. You should see the following:
```

Comment:
It's unclear what these steps are meant to do/what problem they're meant to solve. They need to be contextualized.